api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/BaseHandler.java

```diff
@@ -51,6 +56,17 @@ public Response baseJSON(@Context ServletContext context) {
         return Response.ok(hawkularMetrics).build();
     }
 
+    @GET
+    @Produces({APPLICATION_XHTML_XML, TEXT_HTML})
+    @ApiOperation(value = "Returns some basic information about the Hawkular Metrics service.",
+                  response = String.class, responseContainer = "Map")
+    public void baseHTML(@Context ServletContext context) throws Exception {
+
+        HttpServletRequest request = ResteasyProviderFactory.getContextData(HttpServletRequest.class);
```

Comment:
You should be able to declare `HttpServletRequest` and `HttpServletResponse` params annotated with `@Context`

Then the methods becomes as simple as:
```java
public void baseHTML(@Context HttpServletRequest request, @Context HttpServletResponse response) {
  request.getRequestDispatcher("/static/index.html").forward(request,response);
}
```